cmd/createcluster_internal_test.go

```diff
@@ -189,6 +189,24 @@ func TestCreateCluster(t *testing.T) {
 				return data
 			},
 		},
+		{
+			Name: "test with fee recipient and withdrawal addresses",
```

Comment:
let's add tests for:
* different fee & withdrawal addresses (done)
* same fee & withdrawal addresses
* no fee & withdrawal address

Reply:
These tests are there in this file as part of `TestMultipleAddresses`.